packages/core/admin/ee/admin/pages/SettingsPage/pages/Roles/CreatePage/tests/__snapshots__/index.test.js.snap

```diff
@@ -660,7 +660,9 @@ exports[`<CreatePage /> renders and matches the snapshot 1`] = `
       action="#"
       novalidate=""
     >
-      <div>
+      <div
+        style="height: 0px;"
+      >
```

Comment:
It might be worth checking where this could come from.

Reply:
The height is being calculated and set here: https://github.com/strapi/design-system/blob/5a9b4b4b34eeadcb762aa331727a4f1b944fdb3d/packages/strapi-design-system/src/Layout/HeaderLayout.js#L45

As for the diff, I don't see anything in the release that explains it 🤔 https://github.com/strapi/design-system/releases/tag/v1.6.2

Reply:
I don't see anything, either. Could you check if the create role page continues to work or of this height assignment hides an important element?

Reply:
I confirm the create role page is working with the calculated height 👍